extensions/ql-vscode/src/model-editor/languages/ruby/index.ts

```diff
@@ -37,11 +38,11 @@ export const ruby: ModelsAsDataLanguage = {
         );
         return {
           type: "source",
-          input: "",
```

Comment:
(Minor, just so I understand)
Is removing these input/output fields related to the `endpointType`s? Or is this general clean up?

Reply:
This is just general clean up, but I believe it had to do with `input` and `output` not being in some of the test cases after adding `satisfies`, so it might have been necessary.